source/OutfitInfoDisplay.cpp

```diff
@@ -459,9 +459,13 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 	}
 
 	bool isContinuous = (reload <= 1);
+	double burstReload = outfit.BurstReload();
+	bool isContinuousBurst = (outfit.BurstCount() > 1 && burstReload <= 1 && outfit.TotalLifetime() == 1);
```

Comment:
```suggestion
	bool isContinuousBurst = (outfit.BurstCount() > 1 && outfit.BurstReload() <= 1. && outfit.TotalLifetime() == 1.);
```

Reply:
Should the existing line `bool isContinuous = (reload < 1);` also use a float literal? (`reload` isn't an integer.)

Reply:
oh yeah indeed